<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="REVANASIDDESH" failures="1" tests="5" name="Default test(failed)(failed)" time="0.113" errors="0" timestamp="2023-05-09T13:13:25 IST">
  <testcase classname="com.actitime.testng.CustomerModule" name="CreatCustomer" time="0.016"/>
  <testcase classname="com.actitime.testng.CustomerModule" name="CreatCustomer" time="0.002"/>
  <testcase classname="com.actitime.testng.CustomerModule" name="CreatCustomer" time="0.003"/>
  <testcase classname="com.actitime.testng.CustomerModule" name="ModifyCustomer" time="0.005"/>
  <testcase classname="com.actitime.testng.CustomerModule" name="DeletCustomer" time="0.005">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at com.actitime.testng.CustomerModule.DeletCustomer(CustomerModule.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- DeletCustomer -->
</testsuite> <!-- Default test(failed)(failed) -->
